src/App.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import { Provider } from 'react-redux';
+import { store } from './store';
+import { useAppSelector } from './store/hooks';
+import './App.css';
+import Modal from './componets/Modal';
+import MainPage from './componets/MainPage';
+import UncontrolledForm from './componets/forms/UncontrolledForm/UncontrolledForm';
+import ControlledForm from './componets/forms/ControlledForm/ControlledForm';
+
+function AppContent() {
+  const [activeModal, setActiveModal] = useState<
+    'uncontrolled' | 'controlled' | null
+  >(null);
+  const countries = useAppSelector((state) => state.country.countries);
+
+  const handleCloseModal = () => setActiveModal(null);
+
+  return (
+    <div className="app">
+      <div className="app-header">
+        <h1>Form Management System</h1>
+        <div className="button-group">
+          <button
+            onClick={() => setActiveModal('uncontrolled')}
```

Comment:
You can use currying here 
` onClick={handleChooseForm('uncontrolled')}`
where 
` const handleChooseForm = (form: FormType) => () => setActiveModal(form)`